lms/djangoapps/verify_student/DigiCertSHA2SecureServerCA.pem

```diff
@@ -0,0 +1,58 @@
+#software secure server is not providing the full SSL certificate chain.
```

Comment:
@adeelehsan Hi, 
Just taking a look, I don't think we should commit the `.pem` file. Can you check with devops if that's ok?